_includes/live-demos/comments-callback/example.js

```diff
@@ -0,0 +1,214 @@
+/********************************
+ *   Tiny Comments functions    *
+ * (must call "done" or "fail") *
+ ********************************/
+
+function tinycomments_create(_ref, done, fail) {
```

Comment:
The code formatting in this needs some work, as it's rather non-standard. It'd also be good to use full ES6 since this demo won't work on IE 11. Let me know if you want a hand to fix those issues.

Reply:
I think it's now es6? [`21dcef2` (#2007)](https://github.com/tinymce/tinymce-docs/pull/2007/commits/21dcef22ebffe594c3f83783528e942563187ec1)